x-pack/plugins/observability/public/pages/slo_details/components/error_budget_chart_panel.tsx

```diff
@@ -5,132 +5,106 @@
-          <EuiFlexItem grow={false}>
-            <EuiStat
-              titleColor={isSloFailed ? 'danger' : 'success'}
-              title={numeral(slo.summary.errorBudget.remaining).format(percentFormat)}
-              titleSize="s"
-              description={i18n.translate(
-                'xpack.observability.slo.sloDetails.errorBudgetChartPanel.remaining',
-                { defaultMessage: 'Remaining' }
-              )}
-              reverse
-            />
-          </EuiFlexItem>
-          {errorBudgetTimeRemainingFormatted ? (
-            <EuiFlexItem grow={false}>
-              <EuiStat
-                titleColor={isSloFailed ? 'danger' : 'success'}
-                title={errorBudgetTimeRemainingFormatted}
-                titleSize="s"
-                description={i18n.translate(
-                  'xpack.observability.slo.sloDetails.errorBudgetChartPanel.remaining',
-                  { defaultMessage: 'Remaining' }
-                )}
-                reverse
-              />
-            </EuiFlexItem>
-          ) : null}
-        </EuiFlexGroup>
+      stateTransfer.navigateToWithEmbeddablePackage('dashboards', {
+        state,
+        path,
+      });
+    },
+    [embeddable, slo.id, slo.instanceId]
+  );
 
-        <EuiFlexItem>
-          <WideChart
-            chart="area"
-            id={i18n.translate(
-              'xpack.observability.slo.sloDetails.errorBudgetChartPanel.chartTitle',
-              {
-                defaultMessage: 'Error budget remaining',
-              }
-            )}
-            state={isSloFailed ? 'error' : 'success'}
-            data={data}
-            isLoading={isLoading}
+  return (
+    <>
+      <EuiPanel
+        paddingSize="m"
+        color="transparent"
+        hasBorder
+        data-test-subj="errorBudgetChartPanel"
+        onMouseOver={() => {
+          if (!isMouseOver) {
+            setIsMouseOver(true);
+          }
+        }}
+        onMouseLeave={() => {
+          if (isMouseOver) {
+            setIsMouseOver(false);
+          }
+        }}
+      >
+        <EuiFlexGroup direction="column" gutterSize="l">
+          <ErrorBudgetHeader
+            slo={slo}
+            showTitle={true}
+            isMouseOver={isMouseOver}
+            setDashboardAttachmentReady={setDashboardAttachmentReady}
           />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiPanel>
+
+          <ErrorBudgetChart slo={slo} data={data} isLoading={isLoading} />
+        </EuiFlexGroup>
+      </EuiPanel>
+      {isDashboardAttachmentReady ? (
+        <SavedObjectSaveModalDashboard
+          objectType={i18n.translate(
+            'xpack.observability.slo.item.actions.attachToDashboard.objectTypeLabel',
+            { defaultMessage: 'SLO Error Budget burn down' }
+          )}
+          documentInfo={{
+            title: i18n.translate(
+              'xpack.observability.slo.item.actions.attachToDashboard.attachmentTitle',
```

Comment:
`xpack.observability.slo.item.actions.attachToDashboard.attachmentTitle` seems too generic, should we include `.errorBudgetBurnDown` somewhere to avoid conflcit with auther attachToDashboard actions?